pkg/controlplane/control/manager.go

```diff
@@ -144,6 +144,98 @@ type Manager struct {
 	logger *logrus.Entry
 }
 
+// Restart coredns pods
+func coreDnsRestart(ctx context.Context, m *Manager) error {
```

Comment:
Can the function be part of the manager class?

Reply:
dns.go would be a more natural place for it, IMO. It is not really Manager'ish... 
Also, may want to narrow the scope of parameter passed (e.g., would a Client suffice or do you need the full Manager?)

Reply:
I think I will set it as resolved.